fuse/server.go

```diff
@@ -129,7 +129,7 @@ func (s *fuseServer) Run() error {
 		fuse.DefaultPermissions(),
 	)
 	if err != nil {
-		logrus.Fatal(err)
+		logrus.Error(err)
```

Comment:
This comes from the days we had a single fuse-server in sysbox-fs. We should have relaxed this error the moment we switched to multi-fuse-server ...